core/lensHelpersClassic.ml

```diff
@@ -0,0 +1,139 @@
+open Value
```

Comment:
If this is the old (i.e. pre-ICFP paper/correctness proof) version then I'd be inclined to archive it somewhere but remove from this PR.

If there is a reason to keep it, I'd suggest working out an interface that abstracts over which of LensHelpers and LensHelpersClassic is used, so that no other code winds up depending on anything specific to the implementation.

Reply:
This is the naive lenses implemtentaion which we want to keep (for now) for comparison purposes.